wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -384,6 +396,27 @@ public final class ZMUserSession: NSObject {
         self.contextStorage = contextStorage
         self.recurringActionService = recurringActionService
         self.dependencies = dependencies
+
+        super.init()
+
+        setupAnalyticsSession()
+    }
+
+    private func setupAnalyticsSession() {
+        let analyticsUserProfile: AnalyticsUserProfile = AnalyticsUserProfile(
+            analyticsIdentifier: selfUser.remoteIdentifier.uuidString,
+            teamID: selfUser.membership?.team?.remoteIdentifier?.uuidString,
+            teamRole: TeamRole.stringValue(from: selfUser.teamRole.rawValue),
+            teamSize: selfUser.membership?.team?.members.count,
+            contactCount: selfUser.membership?.team?.members.count
+        )
+
+        if let config = dependencies.analyticsSessionConfiguration {
+            self.analyticsSession = AnalyticsSession(appKey: config.countlyKey, host: config.host, userProfile: analyticsUserProfile)
+            self.analyticsSession?.startSession()
+        } else {
+            self.analyticsSession = nil
```

Comment:
If analyticsSession is optional is should already be nil, which would make the else redundant.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/82ed45aab15ce4770b3b5440bec229fff6e56f75